app-tasks/rf/src/rf/uploads/landsat8/settings.py

```diff
@@ -0,0 +1,7 @@
+"""Settings shared by functions for indexing Landsat 8 data"""
+
+organization = 'dfac6307-b5ef-43f7-beda-b9f208bb7726'
+
+usgs_landsat_url = (
```

Comment:
I think we might need to double-check that the scenes have actually been added to S3 right? This extract and the one on S3 (abbreviated, but at least represents what is on S3) might not match up. Sorry - didn't catch this the first time around.